cdi/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~  Copyright 2017 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.smallrye.config</groupId>
+    <artifactId>smallrye-config-parent</artifactId>
+    <version>1.7.1-SNAPSHOT</version>
```

Comment:
bump to 1.9 ?

Reply:
Why waiting? If you wait you have to do a pull merge again 😄.

Reply:
It is not a breaking change, but it is a change in the structure, so I was thinking that this should go into a major release. I'll probably do a minor release with what we have in master and then maybe we can do this. Most likely the MP Config 2.0 API is still a couple of months away.

Is this something useful for you?

Reply:
Absolutely, as I intend to provide my own CDI implementation based on SmallRyes. The variant will include the option to allow null values instead of the NoSuchElement exception. Currently I would only be able to overwrite the producers via the CDI Specializes annotation, but the AfterDeploymentValidation strikes before that.

Reply:
Ok, lets try to push it fast then :)

Reply:
Bumped to 1.8.2-SNAPSHOT, so we can merge it to master and I'll update everything together on release to 1.9.